_posts/2024-10-19-openloco-v24.10.md

```diff
@@ -0,0 +1,150 @@
+---
+title:      "OpenLoco version 24.10"
+tagline:    "Release blog post"
+author:     "Aaron van Geffen"
+date:       2024-10-19 13:30:00 +0200
+layout:     single
+classes:    wide
+categories: news
+
+gallery_track_clipping:
+  - url: https://github.com/user-attachments/assets/b770518f-2de3-4997-8308-af7acae95652
+    image_path: https://github.com/user-attachments/assets/b770518f-2de3-4997-8308-af7acae95652
+    alt: "Track clipping (before)"
+    title: "Track clipping (before)"
+  - url: https://github.com/user-attachments/assets/b2688c9b-9b96-4cad-add2-95f9a9bb6cf5
+    image_path: https://github.com/user-attachments/assets/b2688c9b-9b96-4cad-add2-95f9a9bb6cf5
+    alt: "Track clipping (after)"
+    title: "Track clipping (after)"
+
+gallery_object_selection:
+  - url: https://github.com/user-attachments/assets/d3759d7a-aca1-4ad7-9554-b5bd2f459537
+    image_path: https://github.com/user-attachments/assets/d3759d7a-aca1-4ad7-9554-b5bd2f459537
+    alt: "Object selection (before)"
+    title: "Object selection (before)"
+  - url: https://github.com/user-attachments/assets/08e2bafe-1fbc-41a9-83ee-bb454985d048
+    image_path: https://github.com/user-attachments/assets/08e2bafe-1fbc-41a9-83ee-bb454985d048
+    alt: "Object selection (after)"
+    title: "Object selection (after)"
+
+gallery_chart_resize:
+  - url: https://github.com/user-attachments/assets/5dec9fe5-4e24-4162-9410-2a715080992a
+    image_path: https://github.com/user-attachments/assets/5dec9fe5-4e24-4162-9410-2a715080992a
+    alt: "In-game charts (before)"
+    title: "In-game charts (before)"
+  - url: https://github.com/user-attachments/assets/db8755b8-375a-46b8-ba3c-8bfc8f492de2
+    image_path: https://github.com/user-attachments/assets/db8755b8-375a-46b8-ba3c-8bfc8f492de2
+    alt: "In-game charts (after)"
+    title: "In-game charts (after)"
+---
+
+OpenLoco v24.10 is out! In this release we've added support for subfolders in
+the object index, reimplemented chart drawing, started work on a new widget system,
+and many other things!
+
+For a complete summary of changes, please find the
+[OpenLoco changelog](https://github.com/OpenLoco/OpenLoco/releases/tag/v24.10) on GitHub.
+
+## First off: assorted bug fixes (#2623, #2625, #2653, #2658)
+
+As usual, we start off with some bug fixes. While we're taking great care not to break the game
+as we're reimplementing bits of it, we're only human, and do make mistakes. Often, it's
+small assumptions that end up not holding.
+
+First on the list is a crash that could occur when attempting to place a track underground.
+Under some circumstances, this could lead to a negative Z coordinate, which is invalid in the game's
+coordinate system. Oops!
+
+Second is a crash that could occur when recalculting a vehicle's reliability. As it turns out,
+this is hard to do if a vehicle doesn't have any cars or parts last, such as when you're selling
```

Comment:
I don't quite understand this. Do you mean the crash occurred when you removed the 'last' vehicle in the train?

Reply:
No, I meant to write 'left' instead of 'last'! Nice catch. Hopefully that clears it up?